docs/core/integration-quality-scale/iqs037.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "IQS037 - Keep the config flow understandable and validate where needed"
+---
+
+IQS037 is as follows:
+
+Keep the config flow understandable and validate where needed
+
+## Reasoning
+
+Not all Home Assistant users are tech-savvy, so we should make sure the config flow is used in an understandable way.
+The things you can do to achieve this are:
+- Use the proper selector at the proper place
+- Validate the input where needed
+- Use `data_description` in the `strings.json` to give context about the input field
```

Comment:
I feel like I want to add more explanation to the points. Should I make subheadings?